Rubberduck.Deployment.Build/Rubberduck.Deployment.Build.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Sunburst.NET.Sdk.WPF/1.0.47">
+  <PropertyGroup>
+    <Product>Rubberduck.Deployment.Build</Product>
+    <Copyright>Copyright © 2018</Copyright>
+    <ProjectGuid>{3A3E3535-FF24-40BD-A4D4-5C88AF6DB1BA}</ProjectGuid>
+    <RootNamespace>Rubberduck.Deployment.Build</RootNamespace>
+    <AssemblyName>Rubberduck.Deployment.Build</AssemblyName>
+    <!-- Installer references depend on the unified output path -->
+    <UnifyOutputPath>true</UnifyOutputPath>
+  </PropertyGroup>
+  <Import Project="..\RubberduckBaseMetaProject.csproj" />
+  <ItemGroup>
+    <Reference Include="Microsoft.Build.Framework" />
+    <Reference Include="Microsoft.Build.Tasks.v4.0" />
+    <Reference Include="Microsoft.Build.Utilities.v4.0">
+      <HintPath>..\..\..\..\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\Microsoft.Build.Utilities.v4.0.dll</HintPath>
```

Comment:
this hintpath seems inappropriate. MSBuild exposes a property `$(MSBuildBinPath)` that should help with this.